modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/Reader.java

```diff
@@ -372,8 +375,13 @@ public OpenAPI read(Class<?> cls,
         // look for field-level annotated properties
         globalParameters.addAll(ReaderUtils.collectFieldParameters(cls, components, classConsumes, null));
 
+        // Make sure that the class methods are sorted for deterministic order
+        // See https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#getMethods--
+        final List<Method> methods = Arrays.stream(cls.getMethods())
+                .sorted(Comparator.comparing(Method::getName))
```

Comment:
According to the linked docs, `Class.getMethods()` does not return methods in any particular order and multiple methods can have the same name... Should some additional properties be added to the comparator?

Reply:
Good point. I will add a custom comparator that also sorts based on the arguments of the method.